test/.env

```diff
@@ -29,7 +37,7 @@ RABBITMQ_ADMIN_PORT=7009
 FTP_PORT=20021
 
 # PIXL Export API
-POSTGRES_HOST=localhost
+POSTGRES_PIXL_DB_HOST=localhost
```

Comment:
I know it wasn't you but should this move? Feels like its not to do with the export api

Reply:
Done L13-14: https://github.com/SAFEHR-data/PIXL/blob/b8d5ec35ce2299018f3f83382be51f8f9379d982/.env.sample#L13-L14